doc/user/content/sql/system-catalog/mz_internal.md

```diff
@@ -314,6 +343,36 @@ messages and additional metadata helpful for debugging.
+<!-- | `sample_rate`           | [`double precision`]         | The sampling rate at the time the execution began.                                                                                                                                                                             | -->
+<!-- | `params`                | [`text list`]                | The values of the prepared statement's parameters.                                                                                                                                                                             | -->
+<!-- | `began_at`              | [`timestamp with time zone`] | The time at which execution began.                                                                                                                                                                                             | -->
+<!-- | `finished_at`           | [`timestamp with time zone`] | The time at which execution ended.                                                                                                                                                                                             | -->
+<!-- | `was_successful`        | [`bool`]                     | Whether the statement executed successfully. Mutually exclusive with `was_canceled` and `was_aborted`.                                                                                                                         | -->
+<!-- | `was_canceled`          | [`bool`]                     | Whether the statement execution was canceled. Mutually exclusive with `was_successful` and `was_aborted`.                                                                                                                      | -->
+<!-- p| `was_aborted`           | [`bool`]                     | Whether the statement execution was aborted. "Aborted" means that the execution had not finished when Materialize restarted (for maintenance or due to an error). Mutually exclusive with `was_successful` and `was_canceled`. | -->
+<!-- | `error_message`         | [`test`]                     | The error returned when executing the statement, or `NULL` if it was successful, canceled or aborted.                                                                                                                          | -->
+<!-- | `rows_returned`         | [`uint8`]                    | The number of rows returned by the statement, if it finished successfully and was of a kind of statement that can return rows, or `NULL` otherwise.                                                                            | -->
+<!-- | `was_fast_path`         | [`bool`]                     | Whether the statement could be answered without building a dataflow (e.g., purely by reading an in-memory index), if it finished successfully and was of a kind of statement that can return rows, or `NULL` otherwise.        | -->
```

Comment:
There's also a third peek mode for constant expressions. This should maybe be a char field similar to the execution status above:

```suggestion
<!-- | `execution_mode`         | [`char`]                     | The execution mode for rows-returning queries. `'f'` (`f`ast path) if the statement could be answered without building a new dataflow (e.g., purely by reading an in-memory index), `'s'` (`s`low path) if a new dataflow was built, `'c'` (`c`onstant) if the statement was a constant expression and no dataflow was needed, or `NULL` if this was not a rows-returning query.        | -->
```